frontend/components/FeatureCard/FeatureCard.stories.mdx

```diff
@@ -0,0 +1,19 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs/blocks";
+import { FeatureCard } from "./FeatureCard";
+
+<Meta title="Layout/FeatureCard" component={FeatureCard} />
+
+# FeatureCard
+
+With `MDX` we can define a story for `FeatureCard` right in the middle of our
+markdown documentation.
+
+export const Template = (args) => <FeatureCard {...args} />;
+
+<Canvas>
+  <Story name="featureCard" args={{ title: "nice" }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable of={FeatureCard} />
```

Comment:
Love this! One improvement I can make here is to import a dummy icon and pass that into the args object, which would greatly help demonstrate this component's beauty.